resources/assets/js/components/Group.jsx

```diff
@@ -296,6 +329,17 @@ export default class Group extends Component {
               <td>
                 {user.observations_count}
               </td>
+              {this.state.isOwner ?
+                <td>
+                  {this.state.leader.id !== user.id ?
+                    <button className="button is-small is-danger"
+                            onClick={() => this._handlePromote.call(this, user)}>
```

Comment:
You don't need to bind `this` to the function if you are already using arrow `=>` functions. In JS, arrow functions auto bind for you.

```suggestion
                            onClick={() => this._handlePromote(user)}>
```